src/test/mgr/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 #scripts
 add_ceph_test(mgr-dashboard-smoke.sh ${CMAKE_CURRENT_SOURCE_DIR}/mgr-dashboard-smoke.sh)
+if(WITH_MGR_DASHBOARD_V2_FRONTEND)
 
-if(WITH_MGR_DASHBOARD_V2_FRONTEND AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64|arm|ARM")
-add_ceph_test(mgr-dashboard_v2-frontend-unittests ${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard_v2/run-frontend-unittests.sh)
-endif(WITH_MGR_DASHBOARD_V2_FRONTEND AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64|arm|ARM")
+  if(WITH_MGR_DASHBOARD_V2_FRONTEND)
+    add_ceph_test(mgr-dashboard_v2-frontend-unittests ${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard_v2/run-frontend-unittests.sh)
+  endif(NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64|arm|ARM")
```

Comment:
This seems like it's probably not intended: `endif` with a different statement than the `if`?  Unless it's cmake trickery that I'm not familiar with.

Didn't you mean to just move the original endif to after the last add_ceph_test line?

Reply:
@jcsp Cut&paste error, will fix